daemon/graphdriver/quota/errors.go

```diff
@@ -3,6 +3,7 @@ package quota // import "github.com/docker/docker/daemon/graphdriver/quota"
 import "github.com/docker/docker/errdefs"
 
 var (
+	//nolint
 	_ errdefs.ErrNotImplemented = (*errQuotaNotSupported)(nil)
```

Comment:
This should be reported upstream

Reply:
Good one; the `nolint` in the end didn't really help I think, so I think I should remove this commit, and create a tracking issue to investigate instead.

I have a feeling it might be due to a combination of build-tags and this anonymous import, but perhaps it's a bug in the linter indeed.

I'll drop this commit